### ISSUE TITLE
Getting warnings like "Parameter [...] is not of type string"

### ISSUE THREAD

**Issue by samuell** [pharmbio/sciluigi#29]
When running sciluigi workflows, or the tests, with a Luigi version later thatn 2.0.1, we get warnings similar to this:

```bash
[samuel sciluigi]$ ./test/run_tests.sh 
NOTICE: this script might not work properly you have CDPATH set to something that contains a folder named 'test'
No handlers could be found for logger "luigi-interface"
/home/samuel/opt/miniconda2/lib/python2.7/site-packages/luigi/parameter.py:259: UserWarning: Parameter NonStringParamWF(instance_name=nonstring_param_wf) is not of type string.
  warnings.warn("Parameter {0} is not of type string.".format(str(x)))
./home/samuel/opt/miniconda2/lib/python2.7/site-packages/luigi/parameter.py:259: UserWarning: Parameter WorkflowTask(instance_name=sciluigi_workflow) is not of type string.
  warnings.warn("Parameter {0} is not of type string.".format(str(x)))
./home/samuel/opt/miniconda2/lib/python2.7/site-packages/luigi/parameter.py:259: UserWarning: Parameter MultiInOutWf(instance_name=sciluigi_workflow) is not of type string.
  warnings.warn("Parameter {0} is not of type string.".format(str(x)))
.
----------------------------------------------------------------------
Ran 3 tests in 0.046s

OK
```

**Comment by apra93** [comment]
Is there an easy way to silence these warnings if there isn't a fix for it yet?

**Comment by samuell** [comment]
Hi @apra93 and others. This one is now fixed in [Version 0.9.7](https://github.com/pharmbio/sciluigi/releases/tag/0.9.7) (also pushed to PyPI).
